packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -0,0 +1,56 @@
+## Amazon Lambda Node.js Library
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> **This is a _developer preview_ (public beta) module. Releases might lack important features and might have
+> future breaking changes.**
+>
+> This API is still under active development and subject to non-backward
+> compatible changes or removal in any future version. Use of the API is not recommended in production
+> environments. Experimental APIs are not subject to the Semantic Versioning model.
+
+---
+<!--END STABILITY BANNER-->
+
+This library provides constructs for Node.js Lambda functions.
+
+### Node.js Function
```

Comment:
For better discoverability, let's put this information in the @aws-cdk/aws-lambda README under "Language-specific APIs" or something like that and here just redirect to the @aws-cdk/aws-lambda module.

Reply:
The whole README?

Reply:
Maybe just a reference to this module from the main module?